src/development/platform-integration/platform-channels.md

```diff
@@ -205,14 +205,15 @@ platform side and vice versa:
 | Map                        | FlValue(FlValue, FlValue) |
 {% endsamplecode %}
 
-## Example: Calling platform-specific Android, iOS, and Windows code using platform channels {#example}
+## Example: Calling platform-specific Android, iOS, Windows, and Linux code using platform channels {#example}
```

Comment:
This heading is getting really long. Should we simplify it?

```suggestion
## Example: Calling platform-specific code using platform channels {#example}
```

Reply:
SGTM